pkg/nack/responder_option.go

```diff
@@ -32,3 +33,14 @@ func ResponderLog(log logging.LeveledLogger) ResponderOption {
 		return nil
 	}
 }
+
+// ResponderRetransmitStats stats of retransmitted packets
+func ResponderRetransmitStats(retransmittedPacketsCount *uint64, retransmittedPacketsBytes *uint64) ResponderOption {
+	*retransmittedPacketsCount = 0
```

Comment:
not sure i understand the reset to 0

Reply:
I just set it to 0 here when initializing the responder instead of setting to it 0 externally (Although it might be already set to 0 externally where the variable is defined, but not explicitly...